diff --git a/android/app/google-services.json b/android/app/google-services.json
index c60bbf15a..cf1d972f0 100644
--- a/android/app/google-services.json
+++ b/android/app/google-services.json
@@ -1,104 +1,39 @@
 {
   "project_info": {
-    "project_number": "184930218130",
-    "firebase_url": "https://api-7231322553409637977-752355.firebaseio.com",
-    "project_id": "api-7231322553409637977-752355",
-    "storage_bucket": "api-7231322553409637977-752355.appspot.com"
+    "project_number": "875282646383",
+    "project_id": "platrum-chat",
+    "storage_bucket": "platrum-chat.appspot.com"
   },
   "client": [
     {
       "client_info": {
-        "mobilesdk_app_id": "1:184930218130:android:a0e553d1a1f043b5",
+        "mobilesdk_app_id": "1:875282646383:android:15c23760606ecf63ed149a",
         "android_client_info": {
-          "package_name": "com.mattermost.react.native"
+          "package_name": "com.platrum.chat"
         }
       },
       "oauth_client": [
         {
-          "client_id": "184930218130-8nahrspll1opff0uogtkh2qsv8coqngo.apps.googleusercontent.com",
+          "client_id": "875282646383-8ju1fabb6b285hrbsjgpvmllf8m3ktop.apps.googleusercontent.com",
           "client_type": 3
         }
       ],
       "api_key": [
         {
-          "current_key": "AIzaSyCkZkU2ECVg-mmdCG5OoHHQXtKiENuvWPE"
+          "current_key": "AIzaSyDNSOYbq0JxJN1nI-W_dWmtYj6HalAPSHw"
         }
       ],
       "services": {
-        "analytics_service": {
-          "status": 2
-        },
         "appinvite_service": {
-          "status": 1,
-          "other_platform_oauth_client": []
-        },
-        "ads_service": {
-          "status": 2
-        }
-      }
-    },
-    {
-      "client_info": {
-        "mobilesdk_app_id": "1:184930218130:android:c7debfa7ea3f75a7",
-        "android_client_info": {
-          "package_name": "com.mattermost.rnbeta"
-        }
-      },
-      "oauth_client": [
-        {
-          "client_id": "184930218130-8nahrspll1opff0uogtkh2qsv8coqngo.apps.googleusercontent.com",
-          "client_type": 3
-        }
-      ],
-      "api_key": [
-        {
-          "current_key": "AIzaSyCkZkU2ECVg-mmdCG5OoHHQXtKiENuvWPE"
-        }
-      ],
-      "services": {
-        "analytics_service": {
-          "status": 2
-        },
-        "appinvite_service": {
-          "status": 1,
-          "other_platform_oauth_client": []
-        },
-        "ads_service": {
-          "status": 2
-        }
-      }
-    },
-    {
-      "client_info": {
-        "mobilesdk_app_id": "1:184930218130:android:2db4058a5b5d6506",
-        "android_client_info": {
-          "package_name": "com.mattermost.rn"
-        }
-      },
-      "oauth_client": [
-        {
-          "client_id": "184930218130-8nahrspll1opff0uogtkh2qsv8coqngo.apps.googleusercontent.com",
-          "client_type": 3
-        }
-      ],
-      "api_key": [
-        {
-          "current_key": "AIzaSyCkZkU2ECVg-mmdCG5OoHHQXtKiENuvWPE"
-        }
-      ],
-      "services": {
-        "analytics_service": {
-          "status": 2
-        },
-        "appinvite_service": {
-          "status": 1,
-          "other_platform_oauth_client": []
-        },
-        "ads_service": {
-          "status": 2
+          "other_platform_oauth_client": [
+            {
+              "client_id": "875282646383-8ju1fabb6b285hrbsjgpvmllf8m3ktop.apps.googleusercontent.com",
+              "client_type": 3
+            }
+          ]
         }
       }
     }
   ],
   "configuration_version": "1"
-}
+}
\ No newline at end of file
diff --git a/app/screens/edit_profile/components/form.tsx b/app/screens/edit_profile/components/form.tsx
index 5d2c105af..b3c0c9c63 100644
--- a/app/screens/edit_profile/components/form.tsx
+++ b/app/screens/edit_profile/components/form.tsx
@@ -93,15 +93,15 @@ const ProfileForm = ({
         return {
             firstName: {
                 ref: firstNameRef,
-                isDisabled: (isSAMLOrLDAP(service) && lockedFirstName) || includesSsoService(service),
+                isDisabled: true,
             },
             lastName: {
                 ref: lastNameRef,
-                isDisabled: (isSAMLOrLDAP(service) && lockedLastName) || includesSsoService(service),
+                isDisabled: true,
             },
             username: {
                 ref: usernameRef,
-                isDisabled: service !== '',
+                isDisabled: true,
             },
             email: {
                 ref: emailRef,
@@ -109,11 +109,11 @@ const ProfileForm = ({
             },
             nickname: {
                 ref: nicknameRef,
-                isDisabled: isSAMLOrLDAP(service) && lockedNickname,
+                isDisabled: true,
             },
             position: {
                 ref: positionRef,
-                isDisabled: isSAMLOrLDAP(service) && lockedPosition,
+                isDisabled: true,
             },
         };
     }, [lockedFirstName, lockedLastName, lockedNickname, lockedPosition, currentUser.authService]);
diff --git a/app/screens/login/form.tsx b/app/screens/login/form.tsx
index 9bf88fd23..92513b211 100644
--- a/app/screens/login/form.tsx
+++ b/app/screens/login/form.tsx
@@ -340,19 +340,6 @@ const LoginForm = ({config, extra, serverDisplayName, launchError, launchType, l
                 endAdornment={endAdornment}
             />
 
-            {(emailEnabled || usernameEnabled) && config.PasswordEnableForgotLink !== 'false' && (
-                <Button
-                    onPress={onPressForgotPassword}
-                    containerStyle={[styles.forgotPasswordBtn, error ? styles.forgotPasswordError : undefined]}
-                    testID='login_form.forgot_password.button'
-                >
-                    <FormattedText
-                        id='login.forgot'
-                        defaultMessage='Forgot your password?'
-                        style={styles.forgotPasswordTxt}
-                    />
-                </Button>
-            )}
             {renderProceedButton}
         </View>
     );
diff --git a/app/screens/server/form.tsx b/app/screens/server/form.tsx
index 192ef4188..beda9df9d 100644
--- a/app/screens/server/form.tsx
+++ b/app/screens/server/form.tsx
@@ -175,8 +175,8 @@ const ServerForm = ({
                     error={urlError}
                     keyboardType='url'
                     label={formatMessage({
-                        id: 'mobile.components.select_server_view.enterServerUrl',
-                        defaultMessage: 'Enter Server URL',
+                        id: 'mobile.components.select_server_view.enterHost',
+                        defaultMessage: 'Enter Host',
                     })}
                     onBlur={onBlur}
                     onChangeText={handleUrlTextChanged}
@@ -190,32 +190,10 @@ const ServerForm = ({
                     value={url}
                 />
             </View>
-            <View style={styles.fullWidth}>
-                <FloatingTextInput
-                    autoCorrect={false}
-                    autoCapitalize={'none'}
-                    enablesReturnKeyAutomatically={true}
-                    error={displayNameError}
-                    label={formatMessage({
-                        id: 'mobile.components.select_server_view.displayName',
-                        defaultMessage: 'Display Name',
-                    })}
-                    onBlur={onBlur}
-                    onChangeText={handleDisplayNameTextChanged}
-                    onFocus={onFocus}
-                    onSubmitEditing={onConnect}
-                    ref={displayNameRef}
-                    returnKeyType='done'
-                    spellCheck={false}
-                    testID='server_form.server_display_name.input'
-                    theme={theme}
-                    value={displayName}
-                />
-            </View>
             {!displayNameError &&
             <FormattedText
-                defaultMessage={'Choose a display name for your server'}
-                id={'mobile.components.select_server_view.displayHelp'}
+                defaultMessage={'You can find out your host from the project link. For example, if your project link is \"https://b1d14c9.platrum.ru\", the host is \"b1d14c9\".'}
+                id={'mobile.components.select_server_view.hostHelp'}
                 style={styles.chooseText}
                 testID={'server_form.display_help'}
             />
diff --git a/app/screens/server/index.tsx b/app/screens/server/index.tsx
index 180ef54c6..03a76440d 100644
--- a/app/screens/server/index.tsx
+++ b/app/screens/server/index.tsx
@@ -217,6 +217,13 @@ const Server = ({
     };
 
     const handleConnect = async (manualUrl?: string) => {
+        var serverUrl = typeof manualUrl === 'string' ? manualUrl : url;
+
+        if ( !serverUrl.startsWith('http') ) {
+            serverUrl =  'https://' + serverUrl + '.chat.platrum.ru';
+            setUrl(serverUrl)
+        }
+
         if (buttonDisabled && !manualUrl) {
             return;
         }
@@ -226,7 +233,6 @@ const Server = ({
             return;
         }
 
-        const serverUrl = typeof manualUrl === 'string' ? manualUrl : url;
         if (!serverUrl || serverUrl.trim() === '') {
             setUrlError(formatMessage(defaultServerUrlMessage));
             return;
@@ -267,6 +273,7 @@ const Server = ({
     const handleUrlTextChanged = useCallback((text: string) => {
         setUrlError(undefined);
         setUrl(text);
+        setDisplayName(text);
     }, []);
 
     const isServerUrlValid = (serverUrl?: string) => {
diff --git a/app/screens/settings/about/about.tsx b/app/screens/settings/about/about.tsx
index 5345308db..4c131ea55 100644
--- a/app/screens/settings/about/about.tsx
+++ b/app/screens/settings/about/about.tsx
@@ -1,25 +1,21 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import Clipboard from '@react-native-clipboard/clipboard';
 import React, {useCallback, useMemo} from 'react';
 import {useIntl} from 'react-intl';
-import {Alert, Text, View} from 'react-native';
+import {Alert, Text, Image, View} from 'react-native';
 import DeviceInfo from 'react-native-device-info';
 
 import Config from '@assets/config.json';
-import Button from '@components/button';
 import CompassIcon from '@components/compass_icon';
 import FormattedText from '@components/formatted_text';
 import SettingContainer from '@components/settings/container';
+import SettingSeparator from '@components/settings/separator';
 import AboutLinks from '@constants/about_links';
-import {SNACK_BAR_TYPE} from '@constants/snack_bar';
 import {useTheme} from '@context/theme';
 import useAndroidHardwareBackHandler from '@hooks/android_back_handler';
 import {t} from '@i18n';
 import {popTopScreen} from '@screens/navigation';
-import {buttonBackgroundStyle, buttonTextStyle} from '@utils/buttonStyles';
-import {showSnackBar} from '@utils/snack_bar';
 import {preventDoubleTap} from '@utils/tap';
 import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 import {typography} from '@utils/typography';
@@ -31,6 +27,7 @@ import Title from './title';
 import TosPrivacyContainer from './tos_privacy';
 
 import type {AvailableScreens} from '@typings/screens/navigation';
+import { Dimensions } from 'react-native';
 
 const MATTERMOST_BUNDLE_IDS = ['com.mattermost.rnbeta', 'com.mattermost.rn'];
 
@@ -84,7 +81,7 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
         footerText: {
             color: changeOpacity(theme.centerChannelColor, 0.64),
             ...typography('Body', 50),
-            marginVertical: 10,
+            marginBottom: 10,
         },
         copyrightText: {
             marginBottom: 0,
@@ -96,17 +93,6 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
         group: {
             flexDirection: 'row',
         },
-        copyInfoButtonContainer: {
-            width: 120,
-            marginTop: 10,
-            position: 'relative',
-        },
-        thinLine: {
-            height: 0.2,
-            backgroundColor: changeOpacity(theme.centerChannelColor, 0.2),
-            alignSelf: 'stretch',
-            marginVertical: 20,
-        },
     };
 });
 
@@ -161,232 +147,108 @@ const About = ({componentId, config, license}: AboutProps) => {
         const buildNumber = config.BuildNumber;
         const version = config.Version;
 
+        let id = t('settings.about.server.version.value');
+        let defaultMessage = '{version} (Build {number})';
+        let values: {version: string; number?: string} = {
+            version,
+            number: buildNumber,
+        };
+
         if (buildNumber === version) {
-            return version;
+            id = t('settings.about.serverVersionNoBuild');
+            defaultMessage = '{version}';
+            values = {
+                version,
+                number: undefined,
+            };
         }
 
-        return intl.formatMessage({id: 'settings.about.server.version.value', defaultMessage: '{version} (Build {buildNumber})'}, {version, buildNumber});
-    }, [config, intl]);
+        return {
+            id, defaultMessage, values,
+        };
+    }, [config]);
 
-    const close = useCallback(() => {
+    useAndroidHardwareBackHandler(componentId, () => {
         popTopScreen(componentId);
-    }, [componentId]);
-
-    useAndroidHardwareBackHandler(componentId, close);
-
-    const copyToClipboard = useCallback(
-        () => {
-            const appVersion = intl.formatMessage({id: 'settings.about.app.version', defaultMessage: 'App Version: {version} (Build {number})'}, {version: DeviceInfo.getVersion(), number: DeviceInfo.getBuildNumber()});
-            const buildNumber = config.BuildNumber;
-            const version = config.Version;
-            const server = buildNumber === version ? intl.formatMessage({id: 'settings.about.server.version.noBuild', defaultMessage: 'Server Version: {version}'}, {version}) : intl.formatMessage({id: 'settings.about.server.version', defaultMessage: 'Server Version: {version} (Build {buildNumber})'}, {version, buildNumber});
-            const database = intl.formatMessage({id: 'settings.about.database', defaultMessage: 'Database: {driverName}'}, {driverName: config.SQLDriverName});
-            const databaseSchemaVersion = intl.formatMessage({id: 'settings.about.database.schema', defaultMessage: 'Database Schema Version: {version}'}, {version: config.SchemaVersion});
-            const copiedString = `${appVersion}\n${server}\n${database}\n${databaseSchemaVersion}`;
-            Clipboard.setString(copiedString);
-            showSnackBar({barType: SNACK_BAR_TYPE.INFO_COPIED, sourceScreen: componentId});
-        },
-        [intl, config],
-    );
+    });
 
     return (
         <SettingContainer testID='about'>
             <View style={styles.logoContainer}>
-                <CompassIcon
-                    color={theme.centerChannelColor}
-                    name='mattermost'
-                    size={88}
-                    testID='about.logo'
-                />
+               <Image
+                        source={require('@assets/images/logo.png')}
+                            style={{height: 180, resizeMode: 'contain',  alignItems: 'center',
+                            flex: 1,
+                            paddingRight: 15,
+                            paddingLeft: 15,
+                        }}
+                    />    
                 <Title
                     config={config}
                     license={license}
                 />
-                <Subtitle config={config}/>
-                <View
-                    style={styles.thinLine}
-                />
+                <SettingSeparator lineStyles={styles.lineStyles}/>
             </View>
-
+            <Subtitle config={config}/>
             <View style={styles.infoContainer}>
                 <View style={styles.group}>
                     <Text
                         style={styles.leftHeading}
                         testID='about.app_version.title'
                     >
-                        {intl.formatMessage({id: 'settings.about.app.version.title', defaultMessage: 'App Version:'})}
+                        {intl.formatMessage({id: 'settings.about.version', defaultMessage: 'App Version:'})}
                     </Text>
                     <Text
                         style={styles.rightHeading}
                         testID='about.app_version.value'
                     >
-                        {intl.formatMessage({id: 'settings.about.app.version.value', defaultMessage: '{version} (Build {number})'},
+                        {intl.formatMessage({id: 'settings.about.build', defaultMessage: '{version} (Build {number})'},
                             {version: DeviceInfo.getVersion(), number: DeviceInfo.getBuildNumber()})}
                     </Text>
                 </View>
-                <View style={styles.group}>
-                    <Text
-                        style={styles.leftHeading}
-                        testID='about.server_version.title'
-                    >
-                        {intl.formatMessage({id: 'settings.about.server.version.title', defaultMessage: 'Server Version:'})}
-                    </Text>
-                    <Text
-                        style={styles.rightHeading}
-                        testID='about.server_version.value'
-                    >
-                        {serverVersion}
-                    </Text>
-                </View>
-                <View style={styles.group}>
-                    <Text
-                        style={styles.leftHeading}
-                        testID='about.database.title'
-                    >
-                        {intl.formatMessage({id: 'settings.about.database.title', defaultMessage: 'Database:'})}
-                    </Text>
-                    <Text
-                        style={styles.rightHeading}
-                        testID='about.database.value'
-                    >
-                        {config.SQLDriverName}
-                    </Text>
-                </View>
-                <View style={styles.group}>
-                    <Text
-                        style={styles.leftHeading}
-                        testID='about.database_schema_version.title'
-                    >
-                        {intl.formatMessage({id: 'settings.about.database.schema.title', defaultMessage: 'Database Schema Version:'})}
-                    </Text>
-                    <Text
-                        style={styles.rightHeading}
-                        testID='about.database_schema_version.value'
-                    >
-                        {config.SchemaVersion}
-                    </Text>
-                </View>
-                <Button
-                    theme={theme}
-                    backgroundStyle={[buttonBackgroundStyle(theme, 'm', 'tertiary'), styles.copyInfoButtonContainer]}
-                    onPress={copyToClipboard}
-                    textStyle={buttonTextStyle(theme, 'm', 'tertiary', 'default')}
-                    text={intl.formatMessage({id: 'settings.about.button.copyInfo', defaultMessage: 'Copy info'})}
-                    testID={'about.copy_info'}
-                    iconName='content-copy'
-                    iconSize={15}
-                    buttonType={'default'}
-                />
-                {license.IsLicensed === 'true' && (
-                    <View style={styles.licenseContainer}>
-                        <FormattedText
-                            defaultMessage='Licensed to: {company}'
-                            id={t('settings.about.licensed')}
-                            style={styles.info}
-                            testID='about.licensee'
-                            values={{company: license.Company}}
-                        />
-                    </View>
-                )}
-                <LearnMore
-                    config={config}
-                    onPress={handleAboutTeam}
-                />
-                {!MATTERMOST_BUNDLE_IDS.includes(DeviceInfo.getBundleId()) &&
+                <View style={[ {marginTop: Dimensions.get('window').height / 5, } ]}>
                     <FormattedText
-                        defaultMessage='{site} is powered by Mattermost'
-                        id={t('settings.about.powered_by')}
-                        style={styles.footerText}
-                        testID='about.powered_by'
-                        values={{site: config.SiteName}}
+                        defaultMessage='© 2022 ООО Платрум, Mattermost Inc.'
+                        id={t('settings.about.copyrightPlatrum')}
+                        style={[styles.footerText, styles.copyrightText, {marginTop: 10}]}
+                        testID='about.copyright'
+                        values={{currentYear: new Date().getFullYear()}}
                     />
-                }
-                <View
-                    style={styles.thinLine}
-                />
-                <FormattedText
-                    defaultMessage='Copyright 2015-{currentYear} Mattermost, Inc. All rights reserved'
-                    id={t('settings.about.copyright')}
-                    style={[styles.footerText, styles.copyrightText]}
-                    testID='about.copyright'
-                    values={{currentYear: new Date().getFullYear()}}
-                />
-                <View style={styles.tosPrivacyContainer}>
-                    <TosPrivacyContainer
-                        config={config}
-                        onPressPrivacyPolicy={handlePrivacyPolicy}
-                        onPressTOS={handleTermsOfService}
-                    />
-                </View>
-                <View style={styles.noticeContainer}>
-                    <FormattedText
-                        id={t('settings.notice_text')}
-                        defaultMessage='Mattermost is made possible by the open source software used in our {platform} and {mobile}.'
-                        style={styles.footerText}
-                        values={{
-                            platform: (
-                                <FormattedText
-                                    defaultMessage='server'
-                                    id={t('settings.notice_platform_link')}
-                                    onPress={handlePlatformNotice}
-                                    style={styles.noticeLink}
-                                />
-                            ),
-                            mobile: (
-                                <FormattedText
-                                    defaultMessage='mobile apps'
-                                    id={t('settings.notice_mobile_link')}
-                                    onPress={handleMobileNotice}
-                                    style={[styles.noticeLink, {marginLeft: 5}]}
-                                />
-                            ),
-                        }}
-                        testID='about.notice_text'
-                    />
-                </View>
-                <View style={styles.hashContainer}>
-                    <View>
-                        <FormattedText
-                            defaultMessage='Build Hash:'
-                            id={t('about.hash')}
-                            style={styles.footerTitleText}
-                            testID='about.build_hash.title'
+                    <View style={styles.tosPrivacyContainer}>
+                        <TosPrivacyContainer
+                            config={config}
+                            onPressPrivacyPolicy={handlePrivacyPolicy}
+                            onPressTOS={handleTermsOfService}
                         />
-                        <Text
-                            style={styles.footerText}
-                            testID='about.build_hash.value'
-                        >
-                            {config.BuildHash}
-                        </Text>
                     </View>
-                    <View>
+                    <View style={styles.noticeContainer}>
                         <FormattedText
-                            defaultMessage='EE Build Hash:'
-                            id={t('about.hashee')}
-                            style={styles.footerTitleText}
-                            testID='about.build_hash_enterprise.title'
-                        />
-                        <Text
+                            id={t('settings.notice_text')}
+                            defaultMessage='Platrum Chat is made possible by the open source software used in our {platform} and {mobile}.'
                             style={styles.footerText}
-                            testID='about.build_hash_enterprise.value'
-                        >
-                            {config.BuildHashEnterprise}
-                        </Text>
+                            values={{
+                                platform: (
+                                    <FormattedText
+                                        defaultMessage='server'
+                                        id={t('settings.notice_platform_link')}
+                                        onPress={handlePlatformNotice}
+                                        style={styles.noticeLink}
+                                    />
+                                ),
+                                mobile: (
+                                    <FormattedText
+                                        defaultMessage='mobile apps'
+                                        id={t('settings.notice_mobile_link')}
+                                        onPress={handleMobileNotice}
+                                        style={[styles.noticeLink, {marginLeft: 5}]}
+                                    />
+                                ),
+                            }}
+                            testID='about.notice_text'
+                        />
                     </View>
                 </View>
                 <View style={{marginBottom: 20}}>
-                    <FormattedText
-                        defaultMessage='Build Date:'
-                        id={t('about.date')}
-                        style={styles.footerTitleText}
-                        testID='about.build_date.title'
-                    />
-                    <Text
-                        style={styles.footerText}
-                        testID='about.build_date.value'
-                    >
-                        {config.BuildDate}
-                    </Text>
                 </View>
             </View>
         </SettingContainer>
diff --git a/app/screens/settings/about/subtitle.tsx b/app/screens/settings/about/subtitle.tsx
index d7bbd2330..c6a9c6d8a 100644
--- a/app/screens/settings/about/subtitle.tsx
+++ b/app/screens/settings/about/subtitle.tsx
@@ -14,8 +14,9 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
         subtitle: {
             color: changeOpacity(theme.centerChannelColor, 0.72),
             ...typography('Heading', 400, 'Regular'),
-            textAlign: 'center',
-            paddingHorizontal: 36,
+            textAlign: 'left',
+            paddingHorizontal: 20,
+            paddingVertical: 10,
         },
     };
 });
diff --git a/app/screens/settings/about/title.tsx b/app/screens/settings/about/title.tsx
index f5a3dbc48..632f27af1 100644
--- a/app/screens/settings/about/title.tsx
+++ b/app/screens/settings/about/title.tsx
@@ -13,9 +13,10 @@ import {typography} from '@utils/typography';
 const getStyleSheet = makeStyleSheetFromTheme((theme) => {
     return {
         title: {
-            ...typography('Heading', 800, 'SemiBold'),
+            ...typography('Heading', 300, 'SemiBold'),
             color: theme.centerChannelColor,
             paddingHorizontal: 36,
+            textAlign: 'left',
         },
         spacerTop: {
             marginTop: 8,
diff --git a/assets/base/i18n/en.json b/assets/base/i18n/en.json
index 6aefb8fb0..cb9b9cf3f 100644
--- a/assets/base/i18n/en.json
+++ b/assets/base/i18n/en.json
@@ -304,9 +304,9 @@
   "emoji_skin.medium_dark_skin_tone": "medium dark skin tone",
   "emoji_skin.medium_light_skin_tone": "medium light skin tone",
   "emoji_skin.medium_skin_tone": "medium skin tone",
-  "extension.no_memberships.description": "To share content, you'll need to be a member of a team on a Mattermost server.",
+  "extension.no_memberships.description": "To share content, you'll need to be a member of a team on a Platrum Chat server.",
   "extension.no_memberships.title": "Not a member of any team yet",
-  "extension.no_servers.description": "To share content, you'll need to be logged in to a Mattermost server.",
+  "extension.no_servers.description": "To share content, you'll need to be logged in to a Platrum Chat server.",
   "extension.no_servers.title": "Not connected to any servers",
   "file_upload.fileAbove": "Files must be less than {max}",
   "find_channels.directory": "Directory",
@@ -478,7 +478,7 @@
   "mobile.calls_limit_msg_GA": "Upgrade to Cloud Professional or Cloud Enterprise to enable group calls with more than {maxParticipants} participants.",
   "mobile.calls_limit_reached": "Participant limit reached",
   "mobile.calls_lower_hand": "Lower hand",
-  "mobile.calls_mic_error": "To participate, open Settings to grant Mattermost access to your microphone.",
+  "mobile.calls_mic_error": "To participate, open Settings to grant Platrum Chat access to your microphone.",
   "mobile.calls_more": "More",
   "mobile.calls_mute": "Mute",
   "mobile.calls_name_is_talking_postfix": "is talking...",
@@ -536,8 +536,10 @@
   "mobile.components.select_server_view.connect": "Connect",
   "mobile.components.select_server_view.connecting": "Connecting",
   "mobile.components.select_server_view.displayHelp": "Choose a display name for your server",
+  "mobile.components.select_server_view.hostHelp": "You can find out your host from the project link. For example, if your project link is \"https://b1d14c9.platrum.ru\", the host is \"b1d14c9\".",
   "mobile.components.select_server_view.displayName": "Display Name",
   "mobile.components.select_server_view.enterServerUrl": "Enter Server URL",
+  "mobile.components.select_server_view.enterHost": "Enter Host",
   "mobile.components.select_server_view.msg_connect": "Let’s Connect to a Server",
   "mobile.components.select_server_view.msg_description": "A server is your team's communication hub accessed using a unique URL",
   "mobile.components.select_server_view.msg_welcome": "Welcome",
@@ -601,7 +603,7 @@
   "mobile.link.error.text": "Unable to open the link.",
   "mobile.link.error.title": "Error",
   "mobile.login_options.cant_heading": "Can't Log In",
-  "mobile.login_options.enter_credentials": "Enter your login details below.",
+  "mobile.login_options.enter_credentials": "Enter Platrum login details below.",
   "mobile.login_options.gitlab": "GitLab",
   "mobile.login_options.google": "Google",
   "mobile.login_options.heading": "Log In to Your Account",
@@ -630,9 +632,9 @@
   "mobile.managed.jailbreak": "Jailbroken devices are not trusted by {vendor}.\n\nReason {reason}\n\n\n\nDebug info: {debug}\n\nPlease exit the app.",
   "mobile.managed.jailbreak_no_debug_info": "Not available",
   "mobile.managed.jailbreak_no_reason": "Not available",
-  "mobile.managed.not_secured.android": "This device must be secured with a screen lock to use Mattermost.",
-  "mobile.managed.not_secured.ios": "This device must be secured with a passcode to use Mattermost.\n\nGo to Settings > Face ID & Passcode.",
-  "mobile.managed.not_secured.ios.touchId": "This device must be secured with a passcode to use Mattermost.\n\nGo to Settings > Touch ID & Passcode.",
+  "mobile.managed.not_secured.android": "This device must be secured with a screen lock to use Platrum Chat.",
+  "mobile.managed.not_secured.ios": "This device must be secured with a passcode to use Platrum Chat.\n\nGo to Settings > Face ID & Passcode.",
+  "mobile.managed.not_secured.ios.touchId": "This device must be secured with a passcode to use Platrum Chat.\n\nGo to Settings > Touch ID & Passcode.",
   "mobile.managed.secured_by": "Secured by {vendor}",
   "mobile.managed.settings": "Go to settings",
   "mobile.markdown.code.copy_code": "Copy Code",
@@ -729,7 +731,7 @@
   "mobile.server_ping_failed": "Cannot connect to the server.",
   "mobile.server_requires_client_certificate": "Server requires client certificate for authentication.",
   "mobile.server_upgrade.button": "OK",
-  "mobile.server_upgrade.description": "\nA server upgrade is required to use the Mattermost app. Please ask your System Administrator for details.\n",
+  "mobile.server_upgrade.description": "\nA server upgrade is required to use the Platrum Chat app. Please ask your System Administrator for details.\n",
   "mobile.server_upgrade.title": "Server upgrade required",
   "mobile.server_url.deeplink.emm.denied": "This app is controlled by an EMM and the DeepLink server url does not match the EMM allowed server",
   "mobile.server_url.empty": "Please enter a valid server URL",
@@ -816,7 +818,7 @@
   "onboarding.realtime_collaboration": "Collaborate in real‑time",
   "onboarding.realtime_collaboration_description": "Persistent channels, direct messaging, and file sharing works seamlessly so you can stay connected, wherever you are.",
   "onboarding.welcome": "Welcome",
-  "onboaring.welcome_description": "Mattermost is an open source platform for developer collaboration. Secure, flexible, and integrated with your tools.",
+  "onboaring.welcome_description": "Platrum Chat is an open source platform for developer collaboration. Secure, flexible, and integrated with your tools.",
   "password_send.description": "To reset your password, enter the email address you used to sign up",
   "password_send.error": "Please enter a valid email address.",
   "password_send.generic_error": "We were unable to send you a reset password link. Please contact your System Admin for assistance.",
@@ -900,7 +902,7 @@
   "rate.error.text": "There has been an error while opening the review modal.",
   "rate.error.title": "Error",
   "rate.subtitle": "Let us know what you think.",
-  "rate.title": "Enjoying Mattermost?",
+  "rate.title": "Enjoying Platrum Chat?",
   "requested_ack.title": "Request Acknowledgements",
   "saved_messages.empty.paragraph": "To save something for later, long-press on a message and choose Save from the menu. Saved messages are only visible to you.",
   "saved_messages.empty.title": "No saved messages yet",
@@ -974,6 +976,7 @@
   "settings.about.app.version.value": "{version} (Build {number})",
   "settings.about.button.copyInfo": "Copy info",
   "settings.about.copyright": "Copyright 2015-{currentYear} Mattermost, Inc. All rights reserved",
+  "settings.about.copyrightPlatrum": "© 2023 ООО Платрум, Mattermost Inc.",
   "settings.about.database": "Database: {driverName}",
   "settings.about.database.schema": "Database Schema Version: {version}",
   "settings.about.database.schema.title": "Database Schema Version:",
@@ -994,10 +997,10 @@
   "settings.link.error.title": "Error",
   "settings.notice_mobile_link": "mobile apps",
   "settings.notice_platform_link": "server",
-  "settings.notice_text": "Mattermost is made possible by the open source software used in our {platform} and {mobile}.",
+  "settings.notice_text": "Platrum Chat is made possible by the open source software used in our {platform} and {mobile}.",
   "settings.notifications": "Notifications",
   "settings.save": "Save",
-  "share_extension.channel_error": "You are not a member of a team on the selected server. Select another server or open Mattermost to join a team.",
+  "share_extension.channel_error": "You are not a member of a team on the selected server. Select another server or open Platrum Chat to join a team.",
   "share_extension.channel_label": "Channel",
   "share_extension.channels_screen.title": "Select channel",
   "share_extension.count_limit": "You can only share {count, number} {count, plural, one {file} other {files}} on this server",
@@ -1090,7 +1093,7 @@
   "unreads.empty.paragraph": "Turn off the unread filter to show all your channels.",
   "unreads.empty.show_all": "Show all",
   "unreads.empty.title": "No more unreads",
-  "unsupported_server.message": "Your server, {serverDisplayName}, is running an unsupported server version. You may experience compatibility issues that cause crashes or severe bugs breaking core functionality of the app. Please contact your System Administrator to upgrade your Mattermost server.",
+  "unsupported_server.message": "Your server, {serverDisplayName}, is running an unsupported server version. You may experience compatibility issues that cause crashes or severe bugs breaking core functionality of the app. Please contact your System Administrator to upgrade your Platrum Chat server.",
   "unsupported_server.title": "Unsupported server version",
   "user_profile.custom_status": "Custom Status",
   "user_status.away": "Away",
diff --git a/assets/base/i18n/ru.json b/assets/base/i18n/ru.json
index 115dc3d7b..654c8fe93 100644
--- a/assets/base/i18n/ru.json
+++ b/assets/base/i18n/ru.json
@@ -304,9 +304,9 @@
   "emoji_skin.medium_dark_skin_tone": "средне-темный оттенок кожи",
   "emoji_skin.medium_light_skin_tone": "средне-светлый оттенок кожи",
   "emoji_skin.medium_skin_tone": "средний оттенок кожи",
-  "extension.no_memberships.description": "Чтобы обмениваться контентом, вам необходимо быть участником команды на сервере Mattermost.",
+  "extension.no_memberships.description": "Чтобы обмениваться контентом, вам необходимо быть участником команды на сервере Platrum Chat.",
   "extension.no_memberships.title": "Пока не является участником какой-либо команды",
-  "extension.no_servers.description": "Чтобы поделиться контентом, вам необходимо войти на сервер Mattermost.",
+  "extension.no_servers.description": "Чтобы поделиться контентом, вам необходимо войти на сервер Platrum Chat.",
   "extension.no_servers.title": "Не подключен ни к одному серверу",
   "file_upload.fileAbove": "Файлы должны быть меньше {max}МБ",
   "find_channels.directory": "Каталог",
@@ -478,7 +478,7 @@
   "mobile.calls_limit_msg_GA": "Обновление до Cloud Professional или Cloud Enterprise позволяет осуществлять групповые звонки с более чем {maxParticipants} участников.",
   "mobile.calls_limit_reached": "Лимит участников достигнут",
   "mobile.calls_lower_hand": "Опустить руку",
-  "mobile.calls_mic_error": "Для участия откройте \"Настройки\", чтобы предоставить Mattermost доступ к вашему микрофону.",
+  "mobile.calls_mic_error": "Для участия откройте \"Настройки\", чтобы предоставить Platrum Chat доступ к вашему микрофону.",
   "mobile.calls_more": "Ещё",
   "mobile.calls_mute": "Отключить звук",
   "mobile.calls_name_is_talking_postfix": "говорит...",
@@ -536,8 +536,10 @@
   "mobile.components.select_server_view.connect": "Подключиться",
   "mobile.components.select_server_view.connecting": "Подключение",
   "mobile.components.select_server_view.displayHelp": "Выберите отображаемое имя для вашего сервера",
+  "mobile.components.select_server_view.hostHelp": "Узнать ваш хост можно по ссылке на проект. Например, если ссылка на ваш проект - \"https://b1d14c9.platrum.ru\", то хостом является \"b1d14c9\".",
   "mobile.components.select_server_view.displayName": "Отображаемое имя",
   "mobile.components.select_server_view.enterServerUrl": "Введите адрес сервера",
+  "mobile.components.select_server_view.enterHost": "Введите Хост",
   "mobile.components.select_server_view.msg_connect": "Давайте подключимся к серверу",
   "mobile.components.select_server_view.msg_description": "Сервер - это коммуникационный узел вашей команды, доступ к которому осуществляется по уникальному URL-адресу",
   "mobile.components.select_server_view.msg_welcome": "Добро пожаловать",
@@ -601,7 +603,7 @@
   "mobile.link.error.text": "Невозможно открыть ссылку.",
   "mobile.link.error.title": "Ошибка",
   "mobile.login_options.cant_heading": "Не могу войти в систему",
-  "mobile.login_options.enter_credentials": "Введите свои данные для входа в систему ниже.",
+  "mobile.login_options.enter_credentials": "Введите вашу почту и пароль от Platrum.",
   "mobile.login_options.gitlab": "GitLab",
   "mobile.login_options.google": "Google",
   "mobile.login_options.heading": "Войдите в свою учетную запись",
@@ -630,9 +632,9 @@
   "mobile.managed.jailbreak": "Устройства с джейлбрейком не являются доверенными {vendor}, выйдите из приложения..\n\nПричина {reason}\n\n\n\nОтладочная информация: {debug}\n\nПожалуйста, выйдите из приложения.",
   "mobile.managed.jailbreak_no_debug_info": "Недоступно",
   "mobile.managed.jailbreak_no_reason": "Недоступно",
-  "mobile.managed.not_secured.android": "Это устройство должно быть защищено с помощью блокировки экрана, чтобы использовать Mattermost.",
-  "mobile.managed.not_secured.ios": "Это устройство должно быть защищено паролем для использования Mattermost. \n\nПерейдите в Настройки > Face ID и пароль.",
-  "mobile.managed.not_secured.ios.touchId": "Это устройство должно быть защищено паролем для использования Mattermost.\n\nПерейдите в Настройки > Touch ID и пароль.",
+  "mobile.managed.not_secured.android": "Это устройство должно быть защищено с помощью блокировки экрана, чтобы использовать Platrum Chat.",
+  "mobile.managed.not_secured.ios": "Это устройство должно быть защищено паролем для использования Platrum Chat. \n\nПерейдите в Настройки > Face ID и пароль.",
+  "mobile.managed.not_secured.ios.touchId": "Это устройство должно быть защищено паролем для использования Platrum Chat.\n\nПерейдите в Настройки > Touch ID и пароль.",
   "mobile.managed.secured_by": "Защищено {vendor}",
   "mobile.managed.settings": "Перейти к настройкам",
   "mobile.markdown.code.copy_code": "Скопировать код",
@@ -729,7 +731,7 @@
   "mobile.server_ping_failed": "Невозможно подключиться к серверу.",
   "mobile.server_requires_client_certificate": "Серверу требуется сертификат клиента для аутентификации.",
   "mobile.server_upgrade.button": "Ок\"",
-  "mobile.server_upgrade.description": "\nДля продолжения работы требуется обновление сервера Mattermost. Пожалуйста, обратитесь к системному администратору за подробностями.\n",
+  "mobile.server_upgrade.description": "\nДля продолжения работы требуется обновление сервера Platrum Chat. Пожалуйста, обратитесь к системному администратору за подробностями.\n",
   "mobile.server_upgrade.title": "Требуется обновление сервера",
   "mobile.server_url.deeplink.emm.denied": "Это приложение управляется EMM, и url сервера DeepLink не соответствует разрешенному серверу EMM",
   "mobile.server_url.empty": "Пожалуйста, введите действительный URL-адрес сервера",
@@ -816,7 +818,7 @@
   "onboarding.realtime_collaboration": "Сотрудничайте в режиме реального времени",
   "onboarding.realtime_collaboration_description": "Постоянные каналы, прямой обмен сообщениями и файлами работают бесперебойно, чтобы вы могли оставаться на связи, где бы вы ни находились.",
   "onboarding.welcome": "Добро пожаловать",
-  "onboaring.welcome_description": "Mattermost - это платформа с открытым исходным кодом для совместной работы разработчиков. Безопасная, гибкая и интегрированная с вашими инструментами.",
+  "onboaring.welcome_description": "Platrum Chat - это платформа с открытым исходным кодом для совместной работы разработчиков. Безопасная, гибкая и интегрированная с вашими инструментами.",
   "password_send.description": "Для сброса пароля введите email адрес, использованный при регистрации",
   "password_send.error": "Пожалуйста, введите корректный email.",
   "password_send.generic_error": "Мы не смогли отправить вам ссылку для сброса пароля. Пожалуйста, обратитесь за помощью к системному администратору.",
@@ -900,7 +902,7 @@
   "rate.error.text": "Произошла ошибка при открытии модального окна обзора.",
   "rate.error.title": "Ошибка",
   "rate.subtitle": "Дайте нам знать, что вы думаете.",
-  "rate.title": "Нравится Mattermost?",
+  "rate.title": "Нравится Platrum Chat?",
   "requested_ack.title": "Запрос подтверждения",
   "saved_messages.empty.paragraph": "Чтобы сохранить сообщение на потом, нажмите на него долгим тапом и выберите в меню пункт Сохранить. Сохраненные сообщения видны только вам.",
   "saved_messages.empty.title": "Нет сохраненных сообщений",
@@ -963,6 +965,7 @@
   "settings.about.app.version.value": "{version} (Сборка {number})",
   "settings.about.button.copyInfo": "Скопировать информацию",
   "settings.about.copyright": "Copyright 2015-{currentYear} Mattermost, Inc. Все права защищены",
+  "settings.about.copyrightPlatrum": "© 2023 ООО Платрум, Mattermost Inc.",
   "settings.about.database": "База данных: {driverName}",
   "settings.about.database.schema": "Версия Схемы Базы Данных: {version}",
   "settings.about.database.schema.title": "Версия Схемы Базы Данных:",
@@ -997,7 +1000,7 @@
   "settings_display.timezone.manual": "Сменить часовой пояс",
   "settings_display.timezone.off": "Выкл",
   "settings_display.timezone.select": "Выберите часовой пояс",
-  "share_extension.channel_error": "Вы не являетесь членом команды на выбранном сервере. Выберите другой сервер или откройте Mattermost, чтобы присоединиться к команде.",
+  "share_extension.channel_error": "Вы не являетесь членом команды на выбранном сервере. Выберите другой сервер или откройте Platrum Chat, чтобы присоединиться к команде.",
   "share_extension.channel_label": "Канал",
   "share_extension.channels_screen.title": "Выбрать канал",
   "share_extension.count_limit": "Вы можете поделиться только {count, number} {count, plural, one {файлом} few{файлами} other {файлами}} на этом сервере",
@@ -1090,7 +1093,7 @@
   "unreads.empty.paragraph": "Отключите фильтр непрочитанных сообщений, чтобы увидеть все ваши каналы.",
   "unreads.empty.show_all": "Показать все",
   "unreads.empty.title": "Больше нет непрочитанных",
-  "unsupported_server.message": "На вашем сервере {serverDisplayName} установлена неподдерживаемая версия сервера. Вы можете столкнуться с проблемами совместимости, которые приведут к сбоям или серьезным ошибкам, нарушающим основную функциональность приложения. Пожалуйста, свяжитесь с вашим системным администратором для обновления вашего сервера Mattermost.",
+  "unsupported_server.message": "На вашем сервере {serverDisplayName} установлена неподдерживаемая версия сервера. Вы можете столкнуться с проблемами совместимости, которые приведут к сбоям или серьезным ошибкам, нарушающим основную функциональность приложения. Пожалуйста, свяжитесь с вашим системным администратором для обновления вашего сервера Platrum Chat.",
   "unsupported_server.title": "Неподдерживаемая версия сервера",
   "user.edit_profile.email.auth_service": "Вход осуществлен через {service}. Адрес электронной почты не может быть обновлен. Адрес электронной почты используемый для уведомлений {email}.",
   "user.edit_profile.email.web_client": "Адрес электронной почты должен изменяться с помощью веб-клиента или настольного приложения.",
